client/components/ManageBotRunDialog/ViewLogsButton/index.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'react-bootstrap';
+
+const ViewLogsButton = ({ externalLogsUrl }) => {
+  const onClick = () => {
+    window.open(externalLogsUrl, '_blank');
+  };
+
+  return (
+    <Button
+      disabled={!externalLogsUrl}
+      aria-disabled={!externalLogsUrl}
+      onClick={onClick}
+      role="link"
+      aria-label={`View logs at ${externalLogsUrl}`}
```

Comment:
```suggestion
      aria-label={`View logs at ${externalLogsUrl}`}
```

This will still say `View logs at null` until the url IS available to a screen reader. Do we want another defaulted statement while it isn't ready?

Alternatively, I'm wondering if announcing the link itself may be unnecessary and the button's description of just "View Log" is sufficient.

Reply:
Good point! I removed it.

Reply:
Thanks for checking that out!